pkg/cluster/audit/audit.go

```diff
@@ -103,7 +103,7 @@ func ShowAuditLog(dir string, auditID string) error {
 		return errors.Trace(err)
 	}
 
-	t := time.Unix(ts, 0)
+	t := time.Unix(ts/1e9, 0)
```

Comment:
This is not compatible with the existing audit files ?

Reply:
Sorry for my careless, @9547 can your please open another PR to fix this?

Reply:
Sorry, I was inconsiderate, I'll fix in another PR

Reply:
fixed in #882 @lucklove @july2993 PTAL